mora/service/common.py

```diff
@@ -93,6 +93,37 @@ def get_connector():
     return lora.Connector(**loraparams)
 
 
+def checked_get(
+    mapping: dict,
+    key: Hashable,
+    default: Hashable,
```

Comment:
I think calling this `default` is misleading, if we don't actually return it as a default value. Maybe this should just be a type instead?